partner_attribute/tests/test_partner_birth_month.py

```diff
@@ -0,0 +1,18 @@
+from odoo.tests.common import TransactionCase
+from datetime import datetime
+from odoo.tools import DEFAULT_SERVER_DATE_FORMAT
+
+class TestPartnerBirthMonth(TransactionCase):
+    def setUp(self):
+        super(TestPartnerBirthMonth, self).setUp()
+        self.partner_admin = self.env.ref("base.partner_admin")
+        self.partner_admin.write({"birthday": "1991-09-05"})
+
+    def test_compute_birth_month(self):
+        self.partner_admin._compute_birth_month()
+        birth_month = int(
+                    datetime.strptime(
+                        str(self.partner_admin.birthday), DEFAULT_SERVER_DATE_FORMAT
+                    ).strftime("%m")
+                )
+        self.assertEqual(self.partner_admin.birth_month, birth_month) 
```

Comment:
This test is pointless if you copy and paste the content of the compute method.  It's not proving anything.

Reply:
When I learn how to write tests from oca repos (https://github.com/OCA/partner-contact/blob/a444fe58d32b7db42fdbb8170be030022ff7fcb4/partner_contact_birthdate/tests/test_res_partner.py#L16-L21), I found some module write tests like this. So, I thought using same content is not the problem.